testsuites/CBLTester/CBL_Functional_tests/conftest.py

```diff
@@ -165,7 +170,7 @@ def params_from_base_suite_setup(request):
                                           port=liteserv_port,
                                           community_enabled=community_enabled,
                                           debug_mode=debug_mode)
-
+    """
```

Comment:
is commenting test server download code, intentional?

Reply:
oops, forgot to remove those quotes. Thanks for catching